Source/WebKit/NetworkProcess/cocoa/NetworkProcessCocoa.mm

```diff
@@ -98,6 +102,12 @@ static void initializeNetworkSettings()
     // Disable NSURLCache.
     auto urlCache(adoptNS([[NSURLCache alloc] initWithMemoryCapacity:0 diskCapacity:0 diskPath:nil]));
     [NSURLCache setSharedURLCache:urlCache.get()];
+
+#if ENABLE(CONTENT_FILTERING_IN_NETWORKING_PROCESS)
+    auto auditToken = *parentProcessConnection()->getAuditToken();
```

Comment:
Is it safe to dereference this `optional<>` here? If the audit token is not available for some reason (perhaps older OS?) this produces an undefined return value.